infrastructure/scripts/create-testdata/stress_testing/create-releases.sh

```diff
@@ -0,0 +1,389 @@
+# usage
+# ./create-releases.sh base-app-name team number-releases
+# Note that this just creates files, it doesn't push in git
+
+BASE_APP_NAME=${1}
+applicationOwnerTeam=${2:-sreteam}
+NUMBER_RELEASES=${3:-100}
+
+# 40 is the length of a full git commit hash.
+authors[0]="urbansky"
+authors[1]="Medo"
+authors[2]="Hannes"
+authors[3]="Mouhsen"
+authors[4]="Tamer"
+authors[5]="Ahmed"
+authors[6]="João"
+authors[7]="Leandro"
+sizeAuthors=${#authors[@]}
+index=$(($RANDOM % $sizeAuthors))
+echo ${authors[$index]}
+author="${authors[$index]}"
+commit_message_file="$(mktemp "${TMPDIR:-/tmp}/publish.XXXXXX")"
+trap "rm -f ""$commit_message_file" INT TERM HUP EXIT
+
+
+msgs[0]="Added new eslint rule"
+msgs[1]="Fix annotations in helm templates"
+msgs[2]="Improve performance with gitLib2"
+msgs[3]="Add colors to new UI"
+msgs[4]="Release trains for env groups"
+msgs[5]="Fix whitespace in ReleaseDialog"
+msgs[6]="Add Snackbar Notifications"
+msgs[7]="Rephrase ReleaseDialog text"
+msgs[8]="Change renovate schedule"
+msgs[9]="Fix bug in distanceToUpstream calculation"
+msgs[10]="Allow deleting locks on locks page"
+sizeMsgs=${#msgs[@]}
+index=$(($RANDOM % $sizeMsgs))
+echo $index
+echo -e "${msgs[$index]}\n" > "${commit_message_file}"
+echo "1: ${msgs[$index]}" >> "${commit_message_file}"
+echo "2: ${msgs[$index]}" >> "${commit_message_file}"
+
+ls "${commit_message_file}"
+
+release_version=''
+case "${RELEASE_VERSION:-}" in
+	*[!0-9]*) echo "Please set the env variable RELEASE_VERSION to a number"; exit 1;;
+	*) release_version='--form-string '"version=${RELEASE_VERSION:-}";;
+esac
+
+
+qaEnvs=()
+configuration+=("--form" "team=${applicationOwnerTeam}")
+search_dir=environments
+for env in "$search_dir"/*
+do
+  env=$(basename "${env}")
+  env=$(echo "$env" | awk '{print tolower($0)}')
+  if [ "$env" != "testing" ]; then
+    qaEnvs+=( "$env" )
+  fi
+done
+
+FRONTEND_PORT=8081 # see docker-compose.yml
+
+for (( c=1; c<=NUMBER_RELEASES; c++ ))
+do
+  deployments=$(($RANDOM % $sizeAuthors))
+  n_deployments=$((10 + $deployments +1))
+  run_release_train=$(($RANDOM % $n_deployments))
+
+  for (( d=1; d<=n_deployments; d++ ))
+  do
+    commit_id=$(head -c 20 /dev/urandom | sha1sum | awk '{print $1}') # SHA-1 produces 40-character hashes
+    inputs=()
+    #inputs+=(--form-string "application=$name")
+    inputs+=(--form-string "source_commit_id=$commit_id")
+    inputs+=(--form-string "source_author=$author")
+    manifests=()
+    search_dir=environments
+
+    for env in "$search_dir"/*
+    do
+      env=$(basename "${env}")
+      env=$(echo "$env" | awk '{print tolower($0)}')
+      signatureFile=$(mktemp "${TMPDIR:-/tmp}/$env.XXXXXX")
+      file=$(mktemp "${x:-/tmp}/$env.XXXXXX")
+      randomValue=$(head -c 20 /dev/urandom | sha1sum | awk '{print $1}' | head -c 12)
+    cat <<EOF > "${file}"
+---
+apiVersion: v1
+data:
+  TNS_ADMIN: /db-config
+  TESTDB_SID_LOOKUP_FILENAME: /db-config/locations.json
+kind: ConfigMap
+metadata:
+  name: foo-service
```

Comment:
this needs to be unique for the cluster/app combination.
Something like this:
```suggestion
  name: foo-service-$app-$env
```